components/function-controller/pkg/apis/serverless/v1alpha2/function_defaults_test.go

```diff
@@ -401,21 +399,13 @@ func TestSetDefaults(t *testing.T) {
 		},
 		"Should ignore label replicas-preset": {
```

Comment:
Maybe we should remove whole this test? IMO without `v1alpha1ReplicasPresetLabel` it has no sense.